ObstacleParams_msg params
---
Obstacle_msg[] obstacles
